client/upgrade/cli/query.go

```diff
@@ -64,3 +65,57 @@ func GetInfoCmd(storeName string, cdc *codec.Codec) *cobra.Command {
 	}
 	return cmd
 }
+
+func GetStatusCmd(storeName string, cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "status",
+		Short:   "query the information of signals",
+		Example: "iriscli upgrade status",
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			cliCtx := context.NewCLIContext().
+				WithCodec(cdc).
+				WithLogger(os.Stdout).
+				WithAccountDecoder(utils.GetAccountDecoder(cdc))
+
+			res_upgradeConfig, err := cliCtx.QueryStore(protocol.UpgradeConfigkey, "protocol")
+			var upgradeConfig protocol.UpgradeConfig
+
+			if err != nil {
+				return err
+			}
+
+			if len(res_upgradeConfig) == 0 {
+				fmt.Println("No Software Upgrade Switch Period is in process.")
+				return err
+			}
+
+			cdc.MustUnmarshalBinaryLengthPrefixed(res_upgradeConfig, &upgradeConfig)
+
+			var validatorAddr []string
+			res, err := cliCtx.QuerySubspace(upgrade.GetSignalPrefixKey(upgradeConfig.Definition.Version), storeName)
+
+			if err != nil {
+				return err
+			}
+
+			for _, kv := range res {
+				validatorAddr = append(validatorAddr, strings.Split(string(kv.Key), "/")[2])
```

Comment:
Add a new util function in upgrade  to impl the `strings.Split(string(kv.Key), "/")[2]`